Commands/Commands/queue.js

```diff
@@ -374,7 +376,7 @@ commands.registerVote = {
                         name: 'f1'
                       }, user.id))
                     })
-                  }, 2500))
+                  }, 500))
```

Comment:
why?

Reply:
I often still see some f1 reactions on suggestions and it's pissing me off

Reply:
They still get removed though, its a 2.5 delay... its not that long.

Reply:
Sometimes they don't, I manually removed around 5 of them like an hour or so ago